doc/api/stream.md

```diff
@@ -540,13 +540,13 @@ Is set to `true` immediately before the [`'finish'`][] event is emitted.
 
 ##### `writable.writableHighWaterMark`
 <!-- YAML
-added: v9.3.0
+added: REPLACEME
```

Comment:
This shouldn't be changed, instead add an additional metadata entry that mentions the setter ability.